gateway/src/lib.rs

```diff
@@ -594,9 +601,11 @@ impl Actor {
     /// - And updated the latest nonce applied for future checks.
     fn apply_msg(rt: &mut impl Runtime, params: ApplyMsgParams) -> Result<RawBytes, ActorError> {
```

Comment:
Let's leave it for now to figure our how we are going to apply bottom up messages, but I think we'll be able to remove this externally-facing method.